jlbh-test.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+mvn clean  test-compile exec:java -Dexec.mainClass="net.openhft.chronicle.wire.TriviallyCopyableJLBH" -Dexec.classpathScope=test
```

Comment:
@j4sm1ne96 @RobAustin please follow the convention of adding to exec plugin and enabling with `run-benchmarks` profile